### PR TITLE
Upgrade to serde 0.8

### PR DESCRIPTION
**Pull request by sfackler**
Also bump ordered-float while we're at it.

### CODE REVIEW

**Comment by sfackler** [comment]
The only real change here is that all of the methods on `Deserializer` are now required.

**Comment by dflemstr** [comment]
Oh sorry, didn't see this PR; I just decided to send a PR for a branch I had laying around for forever.  I am in favor of this change instead of #6

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 /target/
 /Cargo.lock
 /.gitattributes
+/.idea/
+*.iml
```
Comment (dflemstr) [inline .gitignore]: Ignore rules for your local development environment belong in your `~/.gitignore-global` file, IMO. Imagine if we would also add Emacs, Vim, Sublime, ... ignore rules to this file!

Comment (sfackler) [inline .gitignore]: That seems like a not-uncommon thing to do in my experience?

**Comment by arcnmx** [comment]
Thanks, this is now released as 0.3.0!

**Pull request merged**